src/features/areas/components/PublicAreaMap.tsx

```diff
@@ -396,6 +401,36 @@ const PublicAreaMap: FC<PublicAreaMapProps> = ({ area }) => {
           place={selectedPlace}
         />
       )}
+      {isCreating && (
+        <CreatePlaceCard
+          onClose={() => {
+            setIsCreating(false);
+          }}
+          onCreate={(title, type) => {
+            const crosshair = crosshairRef.current;
+            const mapContainer = map?.getContainer();
+            if (crosshair && mapContainer) {
+              const mapRect = mapContainer.getBoundingClientRect();
+              const markerRect = crosshair.getBoundingClientRect();
+              const x = markerRect.x - mapRect.x;
+              const y = markerRect.y - mapRect.y;
+              const markerPoint: [number, number] = [
+                x + 0.5 * markerRect.width,
+                y + 0.8 * markerRect.height,
+              ];
```

Comment:
I think we have enough of these chunks of code now to start thinking about abstracting them away into a utility function. No need to fix for this PR, but whoever touches one of them next should look into it, I think!